index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { ApplicationFunction } from "probot/lib/types";
```

Comment:
I think this should work

```suggestion
import { ApplicationFunction } from "probot";
```

Can you check it please?

Reply:
I tried that at first, but my IDE was saying: `Module '"probot"' has no exported member 'ApplicationFunction'.ts(2305)` And When I checked probot's `index.d.ts` there is only export for `ApplicationFunctionOptions` but no `ApplicationFunction` type.

Reply:
I think we need to update this line first in probot:
https://github.com/probot/probot/blob/bb068d9ae2407df8a07ffa64dc7d7fb8f01f097a/src/index.ts#L24

Reply:
Sure, I'll create PR.

Reply:
Already merged, thanks! Now you'll have to bump the minimum version of probot to `^12.1.4`

https://github.com/probot/adapter-github-actions/blob/8e1f0f86d200547d554fbdbdf01088ec13236fc0/package.json#L34

Note there were some breaking changes in v12, I'm not sure if any of them affect this adapter:
https://github.com/probot/probot/releases/tag/v12.0.0

Reply:
Updated, it seems like these breaking changes doesn't affect adapter, but I found out that there is failing one unit test.

It fails here:

https://github.com/probot/adapter-github-actions/blob/8e1f0f86d200547d554fbdbdf01088ec13236fc0/test/index.test.js#L99

With message:
```js
expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

       97 |     process.stdout.write = origWrite;
       98 |
    >  99 |     expect(output[2].startsWith('::error::{"error":"nope"}%0A{%0A')).toBe(true);
          |                                                                      ^
      100 |
      101 |     expect(mock.activeMocks()).toStrictEqual([]);
      102 |   });

      at Object.<anonymous> (test/index.test.js:99:70)
```

Is it possible that error message format had changed or am I missing something?